test/logflare/lql/rules/select_rule_test.exs

```diff
@@ -0,0 +1,316 @@
+defmodule Logflare.Lql.Rules.SelectRuleTest do
+  use ExUnit.Case, async: true
+
+  alias Logflare.Lql.Rules.SelectRule
+
+  describe "__struct__" do
```

Comment:
do we need these assertions? less code to maintain the better

Reply:
these all get asserted on via other composite/higher-level tests so i would see them as largely redundant